jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735LaneDataAttribute.java

```diff
@@ -7,50 +7,50 @@ public class J2735LaneDataAttribute extends Asn1Object {
 	 * 
 	 */
 	private static final long serialVersionUID = 1L;
-	private Integer pathEndPointAngle;
-	private Integer laneCrownPointCenter;
-	private Integer laneCrownPointLeft;
-	private Integer laneCrownPointRight;
-	private Integer laneAngle;
+	private int pathEndPointAngle;
+	private int laneCrownPointCenter;
+	private int laneCrownPointLeft;
+	private int laneCrownPointRight;
+	private int laneAngle;
```

Comment:
I recall someone requesting for this. I don't remember who. Either Ivan or John. I will ask Ivan what he thinks on this one too.

Reply:
I've probably talked about how I *usually* prefer primitive ints, because they make comparisons etc simpler in Java, and help avoid null reference errors.  But in *this* case these properties are part of a CHOICE data structure, so better to have nullable Integers, since only one at a time will be set, you need the ability for the others to be null.